packages/manager/apps/dedicated/client/app/ip/components/list/firewall/firewall.styles.scss

```diff
@@ -0,0 +1,3 @@
+.overflow-visible {
+  overflow: visible !important;
+}
```

Comment:
**nitpick:** Define the scoped CSS class. Also, add `ip-list-firewall` in HTML template.
```suggestion
.ip-list-firewall{
   .overflow-visible {
     overflow: visible !important;
   }
}
```